Dockerfile

```diff
@@ -4,16 +4,16 @@ WORKDIR /usr/app
 RUN mkdir reports
 RUN pip install poetry
 
-COPY poetry.lock pyproject.toml alembic.ini ./
-
-RUN poetry config virtualenvs.create false
-RUN poetry install --only main --no-root
+COPY poetry.lock pyproject.toml alembic.ini README.md ./
 
 COPY ./src ./src
 COPY ./db_revisions ./db_revisions
 
+RUN poetry config virtualenvs.create false
+RUN poetry install --only main
```

Comment:
no-root basically doesn't install the package, removing that which is needed for package module resolution